File: hitung.py
def hitung_huruf(file_path):
    
    with open(file_path, 'r') as file:

        konten = file.read()

        jumlah_huruf = sum(1 for char in konten if char.isalpha())
    return jumlah_huruf

file_path = input("Masukkan Nama File .txt: ")
jumlah_huruf = hitung_huruf(file_path)
print(f"Jumlah Huruf: {jumlah_huruf}")

File: hitung.php
<?php 
function hitungHuruf($filePath) {
    $konten = file_get_contents($filePath);
    preg_match_all("/[a-zA-Z]/", $konten, $matches);
    return count($matches[0]);
}

echo "Masukkan Nama File .txt: ";
$stdin = fopen('php://stdin', 'r');
$filePath = trim(fgets($stdin));
fclose($stdin);

$jumlahHuruf = hitungHuruf($filePath);
echo "Jumlah Huruf: " . $jumlahHuruf . "\n";
?>